merlin/models/tf/inputs/base.py

```diff
@@ -211,13 +208,13 @@ def InputBlock(
 
 
 def InputBlockV2(
-    schema: Schema,
-    embeddings: Optional[Block] = None,
-    continuous_projection: Optional[Block] = None,
-    continuous_column_selector: Union[Tags, Schema] = Tags.CONTINUOUS,
+    schema: Optional[Schema] = None,
+    categorical: Union[Tags, Schema, Layer] = Tags.CATEGORICAL,
+    continuous: Union[Tags, Schema, Layer] = Tags.CONTINUOUS,
     pre: Optional[BlockType] = None,
     post: Optional[BlockType] = None,
     aggregation: Optional[TabularAggregationType] = "concat",
+    **branches,
```

Comment:
how is this `branches` intended to be used?

Reply:
Good question. I was looking at [this](https://github.com/NVIDIA-Merlin/models/blob/48b7e4ade146052710796e64c8d7f6425fa53e38/tests/unit/tf/blocks/test_interactions.py#L66-L82) test, when we allow for custom branches in the `InputBlockV2`, that could become something like:

```python
InputBlock(
	schema, 
	categorical=CategoryEncoding(
		cat_schema_onehot, sparse=True, output_mode="one_hot"
	),
	multi_hot=CategoryEncoding.multi_hot(cat_schema_multihot, sparse=True)
)
```